RELEASES.md

```diff
@@ -0,0 +1,131 @@
+# 🚀 Releases
+
+This document outlines the Moby project’s release process, including cadence and versioning.
+
+## 📌 Versioning
+
+Releases of Moby modules will be versioned using the dotted triples [Semantic Version](http://semver.org/)
+format. For the purposes of this document, we will refer to the respective
+components of this triple as `<major>.<minor>.<patch>-[suffix[N]]`. The version
+number may have additional information, such as alpha, beta and release candidate
+qualifications along with a numeric identifier. Such releases will be considered
+"pre-releases".
+
+> [!NOTE]
+> v17.03-v20.10 releases were following [CalVer](https://calver.org/).
+> v2.0.0 introduces the `github.com/moby/moby/v2` go module after the
+> previous v28 release of the `github.com/docker/docker` pseudo-module.
+
+## ⏳ Release cadence
+
+The project will release as quickly as possible while ensuring stability and
+timely feature delivery. Maintainers curate a feature roadmap and set milestones
+for each release.
+
+Release dates assigned to milestones are commitments, and any changes will be
+announced to the community via an
+[announcement discussion](https://github.com/moby/moby/discussions/categories/announcements).
+To meet deadlines, some features may be deferred to future releases.
+
+If your issue or feature is missing from the roadmap, please open a GitHub issue
+or comment to request its inclusion.
+
+## 🎯 Major and minor releases
+
+Major and minor releases of moby will be made from main. Releases will be marked
+with GPG signed tags and posted at https://github.com/moby/moby/releases. The
+tag will be of the format `v<major>.<minor>.<patch>-[suffix[N]]`.
```

Comment:
For consistency with previous versions (like https://github.com/moby/moby/releases/tag/v28.3.0-rc.2), let's keep the `.` between the suffix and N:

```suggestion
tag will be of the format `v<major>.<minor>.<patch>-[suffix.[N]]`.
```